en/firefox/developer.ftl

```diff
@@ -8,7 +8,7 @@
 
 firefox-developer-page-title = { -brand-name-firefox-developer-edition }
 firefox-developer-firefox-developer-edition-desc = { -brand-name-firefox-developer-edition } is the blazing fast browser that offers cutting edge developer tools and latest features like CSS Grid support and framework debugging
-firefox-developer-firefox-browser = { -brand-name-firefox-browser } { -brand-name-developer-edition }
+firefox-developer-firefox-browser = { -brand-name-firefox } { -brand-name-developer-edition }
```

Comment:
Note: there are several of these that need new IDs.

Reply:
I guess there's no need to invent the product name like this now, and instead replace all occurences of 
`-firefox-developer-firefox-browser` with just `-brand-name-firefox-developer-edition`…?

Reply:
That's a good call, but at this point relevant for new strings (no point in having these retranslated).